cmd/source_profile.go

```diff
@@ -119,22 +119,22 @@ type SourceProfileConnectionDynamoDB struct {
 func NewSourceProfileConnectionDynamoDB(params map[string]string) (SourceProfileConnectionDynamoDB, error) {
 	// TODO: Populate environment variables for DynamoDB if passed as flags.
 	dydb := SourceProfileConnectionDynamoDB{}
-	if awsAccessKeyID, ok := params["awsAccessKeyID"]; ok {
+	if awsAccessKeyID, ok := params["aws-access-key-id"]; ok {
```

Comment:
do we expect all the keys to be hiphenated ? or is this just the case for AWS ?

Reply:
I was looking at gcloud CLI and noticed that all the flags were hyphenated thus was trying to cleanup and make all the flags and params (keys) hyphenated as well for HB CLI.